cpu/esp32/Makefile.dep

```diff
@@ -173,3 +173,10 @@ endif
 ifneq (,$(filter esp_jtag,$(USEMODULE)))
   FEATURES_REQUIRED += esp_jtag
 endif
+
+# enable bootloader reset over USB, requires CDC ACM to be used
+ifneq (,$(filter usbus_cdc_acm tinyusb_class_cdc,$(USEMODULE)))
```

Comment:
This dependency must also be added to `cpu/esp32/Kconfig.esp32*` for each family :worried: This brings up again what I already have on my ToDo list, a common ESP32x CPU Kconfig model that defines the common properties and dependencies and is used by all families. I will provide a PR in next days. You could either wait for it or add the dependencies to all `cpu/esp32/Kconfig.esp32*` files.

Reply:
If you add it to `cpu/esp32/Kconfig.esp32*`, you can squash it directly. The ACK still applies.

Reply:
It's not that simple, it has to be added to `menuconfig MODULE_TINYUSB_CLASS_CDC` as `imply MODULE_USB_BOARD_RESET` - but only for ESP32?

Reply:
Wouldn't it work simply to use
```
    selcect MODULE_USB_BOARD_RESET if MODULE_STDIO_CDC_ACM || MODULE_TINY_CLASS_CDC
```
in all `CPU_FAM_ESP32x` definitions?

Regardless of this, we still haven't solved the problem that Kconfig doesn't select a STDIO implementation as default for boards with only one USB interface on this board (PR #19038).

Reply:
Ah that also works, thank you!